### ISSUE TITLE
Getting 'Unfortunately Old School Runescape could not be started.' and I don't know why

### ISSUE THREAD

**Issue by randomlama** [TormStorm/jagex-launcher-linux#97]
The relevant configuration and files:
![image](https://github.com/TormStorm/jagex-launcher-linux/assets/7265221/9d0e65a2-e563-4824-90be-3378ccd8656f)
![image](https://github.com/TormStorm/jagex-launcher-linux/assets/7265221/e753bd8e-9100-4a38-9f14-0b5d0756f799)
system info: 
![image](https://github.com/TormStorm/jagex-launcher-linux/assets/7265221/c806d703-5e32-4d49-877c-743c94013d1f)

**Comment by randomlama** [comment]
I followed the manual installation instructions since it didn't seem to work with the install script

**Comment by nmlynch94** [comment]
And this was done using the Lutris script? The location of those files looks non-standard along with the registry value. The Jagex Launcher files are usually inside Program Files x86 by default and changing from that location In the prefix can cause issues. The registry entries assume choosing the default location in the prefix.

**Comment by TormStorm** [comment]
> I followed the manual installation instructions since it didn't seem to work with the install script

If you used the manual installation did you remember to change USERNAME with the name of your user in the windows registry key?

**Comment by randomlama** [comment]
> And this was done using the Lutris script? The location of those files looks non-standard along with the registry value. The Jagex Launcher files are usually inside Program Files x86 by default and changing from that location In the prefix can cause issues. The registry entries assume choosing the default location in the prefix.

I followed the manual installation instructions because the install script didn't work for me, I was unable to type in the jagex launcher so couldn't log in

**Comment by randomlama** [comment]
> > I followed the manual installation instructions since it didn't seem to work with the install script
> 
> If you used the manual installation did you remember to change USERNAME with the name of your user in the windows registry key?

yes I did change it, not sure if you can see my provided screenshots, but it shows my registry key as well

**Comment by nmlynch94** [comment]
@randomlama Try putting your registry key in the Windows format relative to the prefix, something like `C:\users\sebastiaan\AppData\Local\RuneLite`. The unix file path never worked for me in the registry. 

If that doesn't work, I remember what I did was I ran the RuneLite.exe inside the prefix, and then once that was done I just had replaced the RuneLite.exe that the installer placed with the symlink.
